# NavigationRun.msg

time start_time
time end_time
int32 episode_id
